### DISCUSSION TITLE
2024/3テスト版（macOS-12/14テスト、他）

### DISCUSSION THREAD

**Discussion by CSReviser** [CSReviser/CaptureStream2#10] [Announcements]
＜全OS共通＞
（１）カスタムメニューの「保存フォルダ」について「保存フォルダ設定」に加え「保存フォルダ開く」を追加。
　　「保存フォルダ開く」は「保存フォルダ設定」で設定されたフォルダを開きます。
　　※「保存フォルダ」を自分で設定していない方には必要ないかもしれません。
（２）らじる番組（聴き逃し）の情報を取得するプログラムを書き換えました。
（３）一部の番組名の取得方法を変更しました。例：「ｘ曜日」が付く番組名など（ゴガク番組には該当番組はありません）
（４）らじる聴き逃しの番組一覧を表示する、隠し機能を追加しました。
	カスタムメニューの「任意番組設定」内、下部にある「講座一覧表示」にチェックを入れた後、「OK」ボタンをクリック、「スタート」ボタンをクリックするとメッセージウィンドウに一覧が表示されます。
（５）その他bug-fix

＜macOS向け＞
（１）githubでの実行ファイル作成の現行環境（macos-11）廃止に伴う、移行環境のテスト。
　　　・ macos-12：githubでの2024/03現在の標準環境。（CPU：intel）
　　　・ macos-14：githubで2024/04〜06頃に標準環境となる予定。（CPU：Apple-M1）

※MacOS版は動作確認していません。
※今回の更新はテストに協力いただける方向けです。2023/4以降の更新は今回のテスト版をベースにする予定です。

<hr /><em>This discussion was created from the release <a href='https://github.com/CSReviser/CaptureStream2/releases/tag/20240321'>2024/3テスト版（macOS-12/14テスト、他）</a>.</em>

**Comment by nakayamajp** [comment]
macos-14とUniversalを試し、動作に問題ありませんでした。Mac mini M1 + macOS 14.3.1

**Reply by CSReviser** [reply]
動作確認ありがとうございます。
